### PR TITLE
Add jumbo frames and reference clock source parameters to ZIUHFLI

### PR DESCRIPTION
**Pull request by qSaevar**
Changes proposed in this pull request:
- Add parameter to enable TCP/IP jumbo frames
- Add parameter to set the reference clock source
- Fix typo in demod{N}_ oscillator parameters

@WilliamHPNielsen

### CODE REVIEW

File: qcodes/instrument_drivers/ZI/ZIUHFLI.py [hunk 1]
```diff
@@ -1631,6 +1632,25 @@ def __init__(self, name: str, device_ID: str, **kwargs) -> None:
...
+        self.add_parameter('external_clock_source',
+                           set_cmd=partial(self.daq.setInt,
+                                           f"/{self.device}/system/extclk"),
+                           get_cmd=partial(self.daq.getInt,
+                                           f"/{self.device}/system/extclk"),
+                           val_mapping=create_on_off_val_mapping(),
+                           docstring="Set the 10 MHz reference clock source."
+                           )
+
+        self.add_parameter('jumbo_frames',
```
Comment (astafan8) [inline qcodes/instrument_drivers/ZI/ZIUHFLI.py]: perhaps?
```suggestion
        self.add_parameter('jumbo_frames_enabled',
```

File: qcodes/instrument_drivers/ZI/ZIUHFLI.py [hunk 2]
```diff
@@ -1631,6 +1632,25 @@ def __init__(self, name: str, device_ID: str, **kwargs) -> None:
                            parameter_class=Scope,
                            )
 
+        ########################################
+        # SYSTEM PARAMETERS
+        self.add_parameter('external_clock_source',
```
Comment (astafan8) [inline qcodes/instrument_drivers/ZI/ZIUHFLI.py]: perhaps?
```suggestion
        self.add_parameter('external_clock_enabled',
```

File: qcodes/instrument_drivers/ZI/ZIUHFLI.py [hunk 3]
```diff
@@ -1631,6 +1632,25 @@ def __init__(self, name: str, device_ID: str, **kwargs) -> None:
...
+                           set_cmd=partial(self.daq.setInt,
+                                           f"/{self.device}/system/extclk"),
+                           get_cmd=partial(self.daq.getInt,
+                                           f"/{self.device}/system/extclk"),
+                           val_mapping=create_on_off_val_mapping(),
+                           docstring="Set the 10 MHz reference clock source."
```
Comment (astafan8) [inline qcodes/instrument_drivers/ZI/ZIUHFLI.py]: perhaps?
```suggestion
                           docstring="Set the clock source to external 10 MHz reference clock."
```

File: qcodes/instrument_drivers/ZI/ZIUHFLI.py [hunk 4]
```diff
@@ -1631,6 +1632,25 @@ def __init__(self, name: str, device_ID: str, **kwargs) -> None:
                            parameter_class=Scope,
                            )
 
+        ########################################
+        # SYSTEM PARAMETERS
+        self.add_parameter('external_clock_source',
+                           set_cmd=partial(self.daq.setInt,
```
Comment (astafan8) [inline qcodes/instrument_drivers/ZI/ZIUHFLI.py]: shouldn't a value of some other clock-related parameter(s) be updated and/or invalidated once this gets set to True?

Comment (qSaevar) [reply qcodes/instrument_drivers/ZI/ZIUHFLI.py]: No, it has no other effect on the device or device parameters.

**Pull request merged**